packages/adblocker-benchmarks/blockers/cliqz-base.js

```diff
@@ -36,4 +36,12 @@ module.exports = class Cliqz {
       type,
     })).match;
   }
+
+  matchDebug({ url, frameUrl, type }) {
+    return this.engine.match(Request.fromRawDetails({
+      url,
+      sourceUrl: frameUrl,
+      type,
+    })).filter || null;
```

Comment:
Also I think `filter` will be an object and not sure if the `toString()` will be called implicitly somewhere? If not:
```suggestion
    })).filter?.toString() || null;
```